README.md

```diff
@@ -306,6 +306,10 @@ selector function.
 
 This method has 6 overloads.
 
+### HasDuplicates
+
+Checks if an enumerable of T contains duplicates.
+
```

Comment:
Documentation of other methods use `sequence` ; ie: "Check if a sequence contains duplicates elements".

Reply:
good catch!